napari_allencell_annotator/_tests/view/viewer_test.py

```diff
@@ -0,0 +1,113 @@
+from pathlib import Path
+
+import napari
+from bioio import BioImage
+
+import napari_allencell_annotator
+from napari.layers import Points
+
+from napari_allencell_annotator.util.image_utils import ImageUtils
+from napari_allencell_annotator.view.viewer import Viewer, PointsLayerMode
+import numpy as np
+import pytest
+from napari.utils import Colormap
+
+
+@pytest.fixture
+def viewer(make_napari_viewer: napari.Viewer) -> Viewer:
+    return Viewer(make_napari_viewer())
+
+
+def test_all_points_layer(viewer: Viewer) -> None:
+    # ARRANGE
+    viewer.viewer.add_shapes()
+    test_points_layer1: Points = viewer.create_points_layer("test1", "blue", True, 2)
+    test_points_layer2: Points = viewer.create_points_layer("test2", "blue", True, 2)
+
+    # ACT
+    all_points_layer: list[Points] = viewer.get_all_points_layers()
+
+    # ASSERT
+    assert len(all_points_layer) == 2
+    assert test_points_layer1 in all_points_layer
+    assert test_points_layer2 in all_points_layer
+
+
+def test_add_image(viewer: Viewer) -> None:
+    # ARRANGE
+    test_path: Path = (
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "image_types" / "img.ome.tiff"
+    )
+    test_image: BioImage = ImageUtils(test_path).get_image()
```

Comment:
No need to load in this image into memory. We can just initialize a small numpy array (2x2 or so) and add that image instead

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/34583a5cf38b893ef52818fb136db11ad5df4ff6